### DISCUSSION TITLE
Customize Log Output

### DISCUSSION THREAD

**Discussion by Ponjimon** [jmcdo29/ogma#656] [Ideas]
Hello,

is there a way to customize the log output? Like, I don't need the hostname to be logged, for example.

We're running our services in AWS ECS and there I also don't need the timestamp to be logged either. It would be nice if there was a way to change what is actually logged via the Ogma Options e.g. something like this:

```ts
export const OGMA_MODULE_OPTIONS: OgmaModuleOptions = {
  service: {
    color: Boolean(process.env.IS_OFFLINE),
    hostname: false,
    timestamp: false,
  },
  interceptor: {
    http: ExpressParser,
    gql: GraphQLParser,
  },
};
```

**Comment by jmcdo29** [comment]
I'm trying to figure out what would be the best approach here: set up options for each metadata (pid, hostname, timestamp, etc) and do an if to see if it should be added or not, **or** create two new options `formatString` and `formatJSON` to allow for custom log formatting by the end user.
